frog/imports/client/AccountModal/validationHelpers.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+export const errorBasedOnChars = (
+  value: string,
+  x: number,
```

Comment:
NIT: variable should be renamed to `numChar` or similar

Reply:
Fixed